pkgs/games/steam-rom-manager/default.nix

```diff
@@ -0,0 +1,30 @@
+{ appimageTools, lib, fetchurl }:
+let
+  pname = "steam-rom-manager";
+  version = "2.3.40";
+  name = "${pname}";
+
+  src = fetchurl {
+    url = "https://github.com/SteamGridDB/steam-rom-manager/releases/download/v${version}/Steam-ROM-Manager-${version}.AppImage";
+    sha256 = "sha256-qm7F1/+3iVtsxSAptbhiI5sEHR0B9vo7AdEPy1/PANU=";
+  };
+
+  appimageContents = appimageTools.extract { inherit name src; };
```

Comment:
This could be made local to the command:

```
extraInstallCommands = let
  appimageContents = appimageTools.extract { inherit name src; };
  in ''
    ...
  '';
```